agent/profile

```diff
@@ -24,14 +24,13 @@ then
     echo "                Setup has failed - could not determine the proper environment prefix: \"$prefix\"" >&2
     echo "                You will need to fix it before working with pbench." >&2
 else
-    if [[ -f /opt/configtools/bin/getconf.py ]]
+    if [[ -f $PWD/../bin/getconf.py ]]
```

Comment:
Here too, $PWD is going to be problematic. We should use `$prefix` instead. We can assume that the RPM (or whatever takes its place in the future) creates a `common` hierarchy in /opt/pbench-agent (i.e. where $prefix points) and then we can add `$prefix/common/bin` to PATH.

Reply:
ok